### DISCUSSION TITLE
Issues with mapping click-left

### DISCUSSION THREAD

**Discussion by margaretjoanmiller** [polybar/polybar#2682] [Q&A]
Hi! I'm having an issue with binding a click-left action to the network module. Placing `click-left = rofi-wifi-menu.sh &` or any other command under the module config doesn't seem to work because clicking on the network module after polybar is loaded doesn't run the command I want to bind to left-click on that module. What am I missing?

Here's the relevant part of my config file:
```ini
[module/wireless-network]
type = internal/network
click-left = rofi-wifi-menu.sh &
interface = wlo1
interval = 3.0
udspeed-minwidth = 0
format-connected = <label-connected> 
format-disconnected = <label-disconnected>
label-connected = " %essid% "
label-connected-background = ${colors.color1}
label-connected-foreground= ${colors.background}
label-disconnected =  " offline  "
label-disconnected-foreground = ${colors.background}
```

**Comment by patrick96** [comment]
The network module doesn't support any of the `click-*` settings. See https://github.com/polybar/polybar/issues/273#issuecomment-389150884 for a workaround.

**Reply by margaretjoanmiller** [reply]
Thanks! However, I'm a little confused about how to implement action tags in a module. Could you give an in-context example of how they're used?

**Reply by patrick96** [reply]
For example:

```dosini
format-connected = %{A1:rofi-wifi-menu.sh:}<label-connected>%{A1}
```

It's a bit cumbersome, but there are plans to support all click settings in all modules: #1524

**Reply by margaretjoanmiller** [reply]
Oh I see! Thanks!

**Reply by margaretjoanmiller** [reply]
Thanks! That worked!
